src/components/MultiCombobox/MultiCombobox.test.tsx

```diff
@@ -2,35 +2,39 @@ import { renderWithTheme, fireClickAndMouseEvents, fireEvent } from 'test-utils'
 import React from 'react';
 import MultiCombobox, { MultiComboboxProps } from './index';
 
-const ControlledMultiCombobox: React.FC<Partial<MultiComboboxProps<string>>> = props => {
-  const [selectedManufacturer, updateSelectedManufacturer] = React.useState<string[]>([]);
+const items = ['Toyota', 'Ford', 'Chevrolet', 'BMW', 'Mercedes', 'Hammer', 'Dodge', 'Audi'];
+
+const ControlledMultiCombobox = <T,>(props: Partial<MultiComboboxProps<T>>) => {
```

Comment:
I'd suggest simply:

- changing `items`  to be an array of objects
- adding an `itemToString` by default
- In your last test, add an `itemToGroup`

I feel that this way you minimize the amount of changes in this file :)